pkg/kubelet/certificate/bootstrap/bootstrap.go

```diff
@@ -352,6 +352,7 @@ func requestNodeCertificate(client certificatesv1beta1.CertificateSigningRequest
 	ctx, cancel := context.WithTimeout(context.Background(), 3600*time.Second)
 	defer cancel()
 
+	klog.V(2).Infof("Waiting for certificate to be issued")
```

Comment:
Nit: I'm split on whether log level 2 or log level 3 is most appropriate here. I'm looking at https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-output-verbosity-and-debugging for guidance. Could you share a little more about what lead you to chose log level 2?

Reply:
My main rationale is that other logging messages related to bootstrapping statuses are V(2). It would be nice to keep them consistent

See:
pkg/kubelet/certificate/bootstrap/bootstrap.go:64

Reply:
:+1: That makes sense to me.